### PR TITLE
[WIP] Add data processing modules referring to the lassl

### PR DESCRIPTION
**Pull request by gimmaru**
## Title

- add data processing modules referring to the lassl

## Description

- brought data processing functions that fit gpt2 with reference to lassl

## Linked Issues

- None

### CODE REVIEW

File: oslo/data/corpora/kowiki.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+독일의 국기(독일어: Flagge Deutschlands 플라게 도이칠란트[*])는 독일의 국색인 검은색, 빨간색, 금색(Schwarz-Rot-Gold 슈바르츠-로트-골트[*]) 3색의 가로 줄무늬가 같은 비율로 배치된 삼색기이다.[1] 이 기는 바이마르 공화국 시대였던 1919년에 처음 국기로 지정되었다. 별칭은 연방기(Bundesflagge 분데스플라게[*])이다.[2]
```
Comment (hyunwoongko) [inline oslo/data/corpora/kowiki.txt:1]: it seems for the test. right?
when we merge this PR, we should remove this.

File: oslo/data/serialize_corpora.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+from src.processors import GPT2Processer
+from src.utils import load_corpora
+
+model_type_to_processor = {
+    "gpt2": GPT2Processer,
+}
+
+
+@dataclass
+class Arguments:
```
Comment (hyunwoongko) [inline oslo/data/serialize_corpora.py:15]: same. I think this class should not be included under `tests` directory of root.

File: oslo/data/src/collators.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+def tolist(x):
+    if isinstance(x, list):
+        return x
+    elif hasattr(x, "numpy"): # Checks for TF tensors without needing the import
+        x = x.numpy()
+    return x.tolist()
+
+
+class DataCollatorForGpt2:
```
Comment (hyunwoongko) [inline oslo/data/src/collators.py:17]: Isn't `DataCollatorForGPT2` better?

Comment (hyunwoongko) [reply oslo/data/src/collators.py:17]: I think we can split collator source file and move into each model directory.
as you see the `oslo.transforers.models`, there are `gpt2`directory.
how about renaming `collator_gpt2.py` and moving to this directory?

**Comment by hyunwoongko** [comment]
And, eventually this should work with `oslo/transformers/trainer`, so it should be inside `oslo/transformers`. We cannot import this code from the `oslo/transformers` folder if it is in an external folder called `data`.

**Comment by hyunwoongko** [comment]
When you are all ready to merge, please tell me :)

**Comment by hyunwoongko** [comment]
@gimmaru how about merging this? and let's continue working.

**Comment by gimmaru** [comment]
Yes, if there is a problem, I will correct it right away. And I'll continue with the rest of the work

**Comment by hyunwoongko** [comment]
@gimmaru Do not merge yourself. Merging is done by me.

**Comment by gimmaru** [comment]
I'm sorry. I'll be careful from now on

**Pull request merged**